_elements/buttons.md

```diff
@@ -8,26 +8,55 @@ title: Buttons
   <!-- Add HTML markup for example here -->
 
   <h3>Primary Buttons</h3>
-  <button>Static</button>
-  <button class="usa-button-active">Active</button>
-  <button class="usa-button-hover">Hover</button>
+  <div class="button_wrapper">
+    <button>Sign up</button>
+    <button class="usa-button-active">Sign up</button>
```

Comment:
Can you add back in the button text which states it’s current state (static, hover, active)? Without this, users will not know what the 3 colors are for.